src/deluge/gui/ui/menus.h

```diff
@@ -85,6 +85,8 @@ extern MenuItem* paramShortcutsForAudioClips[15][8];
 extern MenuItem* paramShortcutsForSongView[15][8];
 extern MenuItem* paramShortcutsForKitGlobalFX[15][8];
 
+extern deluge::gui::menu_item::Submenu* parentsForSoundShortcuts[15][8];
```

Comment:
same nit here about trying to replace [8] and [15] with kDisplayHeight and kDisplayWidth respectively